spec/integrations/posts_show_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe 'posts show', type: :system do
```

Comment:
- [ ] I think one test is left:
- `Tests that asserts you can see the username of each commentor.`